main.tf

```diff
@@ -308,7 +346,7 @@ locals {
 }
 
 resource "aws_ecs_service" "main" {
-  count = "${var.associate_lb}"
+  count = "${var.associate_alb || var.associate_nlb ? 1 : 0}"
```

Comment:
Does this really need the `? 1: 0`? These are both bools. I think it makes it extra clear this is different from the one below.